tests/unit/test_image_builder.py

```diff
@@ -52,3 +55,14 @@ def test_create_image():
         "https://copr.fedoraproject.org/coprs/foo/bar/",
     )
     assert response == "foo-baz-bar"
+
+
+def test_refresh_fails():
+    """Test that refreshing with no token fails properly"""
+    flexmock(ImageBuilder).should_receive("_get_access_token").and_return(None)
+    with pytest.raises(PackitException) as exc:
+        ImageBuilder(None)
```

Comment:
What's this supposed to do?

Reply:
"Test that refreshing with no token fails properly"

I'll provide more context as a comment in the test